### ISSUE TITLE
Supports time zone setting

### ISSUE THREAD

**Issue by killme2008** [GreptimeTeam/greptimedb-ingester-erl#35]
Related to https://github.com/GreptimeTeam/greptime-proto/pull/132

**Comment by fengjiachun** [comment]
‌‌‌It seems that the time zone is used in the query. Does this client have query method?

**Comment by killme2008** [comment]
> ‌‌‌It seems that the time zone is used in the query. Does this client have query method?

No, the erlang client doesn't have the query functions, so this feature may be not valuable currently. I just add a note here.
